lib/truncate.js

```diff
@@ -8,123 +9,141 @@ exports.span = truncSpan
 exports.error = truncError
 
 function truncMetadata (metadata, opts) {
-  metadata.process.title = truncate(String(metadata.process.title), opts.truncateKeywordsAt)
+  breadthFilter(metadata, (value, key, path) => {
+    if (typeof value !== 'string') {
+      return value
+    }
+
+    let max = opts.truncateStringsAt
+    if (path.join('.') === 'process.title') {
```

Comment:
Would it be less expensive to look at `path[0]` and `path[1]` individually?

Reply:
fixed